INSIGHTS-ONPREM.md

```diff
@@ -0,0 +1,34 @@
+# Insights OnPrem instructions for starting koku in development environment
+## Kafka
+We use the ros-ocp-backend kafka container. Start the ros-ocp-backend kafka service.
+
+## Build and start
+- copy .env.example to new .env file
+- execute `docker-compose build`
+- execute `docker-compose up -d`. some services fail the first time - restart them
```

Comment:
curious, do you know why they fail? I'd be an extra help to show what success looks like for this command for the first timers. WDYT?

Reply:
I do not know why they fail. I know it is due to permissions. but why the same permissions have different effect after restarting I do not know yet. I think the order of the services will also need to change. once we start working on adding koku officially , I can fix it.